_amenities/pools/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Klahanie Pools

```

Comment:
A couple questions about the expanding nav bar: (1) Shouldn't the small directional arrow next to Amenities (and other nav bar elements) change direction when that item is selected? For instance, I'm used to seeing the arrow pointing to the right as a default and then change to pointing down when selected/expanded. (2) Currently, when a user clicks "Amenities," the Amenities menu appears, but the Pool Information sub menu is already expanded. Is this correct? Seems like Pool Information should not be expanded by default?